### PR TITLE
feat: add kizeo forms

### PR DESCRIPTION
**Pull request by BastienMe**
## What does this PR do?

Add new piece : Kizeo Forms

### CODE REVIEW

File: packages/pieces/kizeo-forms/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { downloadCustomExportInItsOriginalFormat } from "./lib/actions/download-custom-export-in-its-original-format";
+import { eventOnDataDeleted } from "./lib/trigger/event-on-data-deleted";
+import { eventOnDataFinished } from "./lib/trigger/event-on-data-finished";
+import { eventOnDataPushed } from "./lib/trigger/event-on-data-pushed";
+import { eventOnDataPulled } from "./lib/trigger/event-on-data-received";
+import { eventOnDataUpdated } from "./lib/trigger/event-on-data-updated";
+
+export const kizeoForms = createPiece({
+  displayName: "Kizeo Forms",
+  logoUrl: "https://cdn.zapier.com/storage/developer_cli/84d326e004b57e6afa863f3ed6c2828a.png",
```
Comment (abuaboud) [inline packages/pieces/kizeo-forms/src/index.ts]: Can you please replace with `https://cdn.activepieces.com/pieces/kizeo-forms.png`

File: packages/pieces/kizeo-forms/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  authors: [],
```
Comment (abuaboud) [inline packages/pieces/kizeo-forms/src/index.ts]: Can you add your github handle there?

**Review by abuaboud — CHANGES_REQUESTED** [review]

**Comment by abuaboud** [comment]
Thank you for your contribution!

@BastienMe, could you please explain how you plan to use it?

I haven't tested the piece thoroughly. Can you please confirm that you have tested it?

**Comment by abuaboud** [comment]
@allcontributors please add @BastienMe for plugin

**Comment by abuaboud** [comment]
> Thank you for your contribution!
> 
> @BastienMe, could you please explain how you plan to use it?
> 
> I didn't test the piece and I am not familiar with kizeo forms. Can you please confirm that you have tested it?

@BastienMe Approved, but can you please confirm and I will merge after.

**Review by abuaboud — APPROVED** [review]

**Comment by BastienMe** [comment]
> > Thank you for your contribution!
> > @BastienMe, could you please explain how you plan to use it?
> > I didn't test the piece and I am not familiar with kizeo forms. Can you please confirm that you have tested it?
> 
> @BastienMe Approved, but can you please confirm and I will merge after.

Hello @abuaboud, thank you for your message.

Kizeo Forms is a mobile forms application that allows the user to automate various activities and transform paper documents into digital equivalents. It offers a wide range of features suitable for any industry, making it a versatile data collection tool.

The piece allows the user to 

- Retrieve the definition of data entered in a form (with a trigger or manually):
  This functionality enables users to retrieve the details and structure of data entered in a Kizeo Forms form. In this way, they can obtain information on the fields, options and parameters associated with this data.

- Send pushed data to a Kizeo Forms mobile application user:
  With this functionality, users can send a pre-filled form to users of the Kizeo Forms mobile application.

- Download an export of data entered in a form according to the format configured by the user:
  Kizeo Forms allows users to configure an export format such as PDF, Word or Excel, this feature allows users to download an export in the format of their choice.
  
I confirm that I have tested it. The results of my tests have been positive, and the piece appears to meet my required specifications.

**Review by abuaboud — APPROVED** [review]

**Pull request merged**